docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -1,20 +1,33 @@
 ---
 id: deploy-process-application
-title: Deploy and run your process application
-description: You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+title: Validate and deploy your process application
+description: Validate your process application in development before deploying it to testing, staging, or production.
 ---
 
 import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You should validate your process application in development before deploying it to testing, staging, or production.
```

Comment:
Agreed with the first one - technically this is more of a strong recommendation as it assumes they follow the dev lifecycle we want them to (hence should rather than must), but I think your suggestion works - thank you 👍